namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-public-production/resources/route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "route53_zone" {
+  name = "find-legal-advice.justice.gov.uk"
```

Comment:
This zone already exists in CP account.

Reply:
Oops, copy and paste fail! Thanks

Reply:
I might put that in a variable next time... As I'd got a bit complacent that that file wasn't hardcoding stuff 🤦‍♂